tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -587,10 +577,11 @@ private boolean testWorkflow(Tool tool) {
             return false;
         }
         Long entryId = workflow.getId();
-        for (String runner : this.runner) {
+        for (String runner : tooltesterConfig.getRunner()) {
             List<WorkflowVersion> verifiedVersions = workflow.getWorkflowVersions().stream().filter(version -> version.isVerified())
                     .collect(Collectors.toList());
-            List<WorkflowVersion> verifiedAndNotBlacklistedVersions = verifiedVersions.stream().filter(version -> BlackList.isNotBlacklisted(toolId, version.getName())).collect(Collectors.toList());
+            List<WorkflowVersion> verifiedAndNotBlacklistedVersions = verifiedVersions.stream()
```

Comment:
This happens often enough, maybe should have a function for it just to stop possible typos and whatnot.

Reply:
Different types, different values